### ISSUE TITLE
before_11286_b in V H.1

### ISSUE THREAD

**Issue by gerritbruening** [faustedition/xml#134]

**Comment by gerritbruening** [comment]
11286f. erscheint zweimal als eingefügt.

**Comment by dietmarpravida** [comment]
scheint jetzt okay, oder?

**Comment by gerritbruening** [comment]
Nein, in der letzten Stufe ist die Umstellung nicht berücksichtigt:
![grafik](https://user-images.githubusercontent.com/10358005/46409065-7326dc00-c714-11e8-9671-9120518fab7c.png)

**Comment by gerritbruening** [comment]
Teil von https://github.com/faustedition/faust-gen-html/issues/627#issuecomment-443226548.
